windows-installer/generate_sdl_agent_exe.ps1

```diff
@@ -143,10 +150,14 @@ $env:Path = [System.Environment]::GetEnvironmentVariable("Path","Machine") + ";"
 # unneeded docs that bloat the file size (and also seem to cause issues with unzipping).
 ###############################
 
-& $GEM_BAT update --system 2.4.8
-& $GEM_BAT install fluentd:0.14.1 --no-ri --no-rdoc --no-document
+# Install a specific version of serverengine with fluentd as there is a compatibility issue with later versions
```

Comment:
We probably will.  Changing nothing this error came up due to this bug, when I rebuilt this..